client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -224,7 +285,11 @@ class StatModuleChartTabs extends Component {
 				<StatsModulePlaceholder className="is-chart" isLoading={ isActiveTabLoading } />
 
 				{ chartType === 'bar' ? (
-					<Chart barClick={ this.props.barClick } data={ chartData } minBarWidth={ 35 }>
+					<Chart
+						barClick={ this.props.barClick }
+						data={ transformBarChartData( chartData, this.props.activeLegend ) }
```

Comment:
I don't quite understand why we changed the shape of the data in `buildChartData` and then transform the data even for bar chart 🤔

Reply:
I agree - transforming for one is what we want. In the end, I got things working with two transformations but I will take another hit at it now with fresh eyes and see if I can cut it back to just the one.